cirq_superstaq/custom_gates.py

```diff
@@ -206,10 +206,22 @@ def __repr__(self) -> str:
 CR = ZX = ZXPowGate()  # standard CR is a full turn of ZX, i.e. exponent = 1
 
 
+@cirq.value_equality
 class AceCR(cirq.Gate):
-    def __init__(self, polarity: str) -> None:
-        assert polarity in ["+-", "-+"]
+    """Active Cancellation Echoed Cross Resonance gate, supporting polarity switches and sandwiches.
+
+    The typical AceCR in literature is a positive half-CR, then X on control, then negative half-CR.
+    Args:
+        polarity: should be either "+-" or "-+". Specifies if positive or negative half-CR is first
+        sandwich_rx_rads: during the X on control, an rx gate can be simultaneously
+            applied to the target.
+    """
+
+    def __init__(self, polarity: str, sandwich_rx_rads: Optional[cirq.value.TParamVal] = 0) -> None:
```

Comment:
```suggestion
    def __init__(self, polarity: str, sandwich_rx_rads: cirq.TParamVal = 0) -> None:
```
(`Optional[A]` is just an alias for `Union[A, None]`, so not necessary here)